SwiftUI.swift

```diff
@@ -32343,6 +32346,54 @@ extension PrimitiveButtonStyle {
 }
 
 /// The properties of a button.
+///
+/// The ``PrimitiveButtonStyle`` definition is very similar to that of
+/// ``ButtonStyle``, except that in the case of the former, we pass a
+/// `PrimitiveButtonStyleConfiguration` to the ``PrimitiveButtonStyle/makeBody(configuration:)``.
+/// function. `PrimitiveButtonStyleConfiguration` comes with the button
+/// `label` as a property, but replaces the ``buttonStyleConfiguration/isPressed``
+/// property with a ``PrimitiveButtonStyleConfiguration/trigger()`` function.
+///
+/// Here, we create a custom `TripleTapOnlyStyle`, which has a
+/// ``View/onTapGesture(count:perform:)`` added to the label to trigger the
+/// button action with three consecutive taps.
+///
+/// ```
+/// struct TripleTapOnlyStyle: PrimitiveButtonStyle {
+///     func makeBody(configuration: Configuration) -> some View {
+///         configuration.label
+///             .onTapGesture(count: 3,
```

Comment:
Two options:
```suggestion
///             .onTapGesture(count: 3) { configuration.trigger }
```

or

```suggestion
///             .onTapGesture(count: 3, perform: configuration.trigger)
```